rasa/model_training.py

```diff
@@ -590,7 +590,9 @@ async def _train_core_with_validated_data(
                 await rasa.core.train.train_in_chunks(
```

Comment:
Do we want to support finetuning?
It's okay to leave it out in case this is going to be a prototype, but i think you will just need to add a `model_to_finetune` parameter to this method and set the agent's ensemble to `model_to_finetune`'s policy ensemble and the rest should be fine.

Reply:
my thinking was: if you do fine-tuning, you don't fine-tune on a lot of data, so normal train should be good